financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/ManifestExtensions.kt

```diff
@@ -3,6 +3,14 @@ package com.stripe.android.financialconnections.features.common
 import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest
 import com.stripe.android.financialconnections.model.SynchronizeSessionResponse
 
+private const val EMAIL_LENGTH = 15
+
+private val FinancialConnectionsSessionManifest.shouldAttachLinkedPaymentMethod: Boolean
+    get() = paymentMethodType != null
+
+internal val FinancialConnectionsSessionManifest.canSaveAccountsToLink: Boolean
+    get() = accountholderIsLinkConsumer == true && isNetworkingUserFlow == true && !shouldAttachLinkedPaymentMethod
```

Comment:
Maybe we can rename this to something more specific? like `shouldSaveAccountsOnSelection` (really bad naming probably, but something on those lines)

On payment flows, accounts get saved to Link on the `AttachPaymentMethod` via the `attach_payment_account` endpoint (technically accounts can be saved to link, but this would be false)

Reply:
Tweaked the naming in the latest commit.